authoring/tools-for-developers.md

```diff
@@ -0,0 +1,80 @@
+# Tools for developers
+
+This section presents some useful tools that can be very helpful for the development workflow.
+
+
+## Code linters
+
+Code linteres are tools that analyze source code and identify programming errors, stylistic errors,
+invalid typing, wrong or missing docstrings, etc.
+
+Some linters that could help you to keep your code in good shape are:
+
+- [flake8](https://flake8.pycqa.org/en/latest/)
+- [mypy](http://mypy-lang.org/)
+- [numpydoc](https://numpydoc.readthedocs.io/en/latest/)
+- [pydocstyle](https://github.com/PyCQA/pydocstyle)
+
+
+## Code stylers
+
+Code stylers are tools that fix styling issues in a file, formatting it automatically. 
+Using an automatic formatting tool helps to keep the source code within specification
+and also helps review workflow.
+
+Some popular code stylers are:
+
+- [black](https://github.com/psf/black)
```

Comment:
same here - maybe we just recommend black and link to `isort` in a "see also" block below? Alternatively, we could add a sentence about what each one of them does, so that users know how they might complement one another etc